quickcheck-dynamic/src/Test/QuickCheck/StateModel.hs

```diff
@@ -96,31 +96,31 @@ class
   --
   -- @
   --   data Action RegState a where
```

Comment:
This whole comment would need to be updated to deal with the added type parameter